inc/namespace.php

```diff
@@ -191,6 +191,13 @@ function disable_emojis_remove_dns_prefetch( array $urls, string $relation_type
 	// Strip out any URLs referencing the WordPress.org emoji location.
 	$emoji_svg_url_bit = 'https://s.w.org/images/core/emoji/';
 	foreach ( $urls as $key => $url ) {
+		if ( is_array( $url ) ) {
+			if ( isset( $url['href'] ) ) {
+				$url = $url['href'];
+			} else {
+				continue;
+			}
```

Comment:
I'd probably flip this to focus on the happy path and reduce nesting.
```suggestion
			if ( ! isset( $url['href'] ) ) {
				continue;
			}

			$url = $url['href'];
```